src/Illuminate/Database/Eloquent/Concerns/HasAttributes.php

```diff
@@ -883,6 +883,23 @@ public function getOriginal($key = null, $default = null)
     }
 
     /**
+     * Get a subset containing the provided attributes with values from the model.
+     * 
+     * @param  array|mixed  $attributes
+     * @return array
+     */
+    public function only($attributes)
+    {
+        $results = [];
+
+        foreach (is_array($attributes) ? $attributes : func_get_args() as $attribute) {
+            $results[$attribute] = $this->getAttribute($attribute);
+        }
+
+        return $results;
```

Comment:
Used the same style as `Request::only` https://github.com/laravel/framework/blob/78fedf79e41cc11e2608e3a06433d5b7c50962ed/src/Illuminate/Http/Concerns/InteractsWithInput.php#L144